src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -349,6 +350,26 @@ public void sendLocalNodeRequest(TransportService transportService) {
         }
     }
 
+    /**
+     * Requests the ActionListener onFailure method to be run by OpenSearch.  The result will be handled by a {@link ActionListenerOnFailureResponseHandler}.
+     *
+     * @param transportService  The TransportService defining the connection to OpenSearch.
+     */
+    public void sendActionListenerOnFailureRequest(TransportService transportService) {
+        logger.info("Sending ActionListener onFailure request to OpenSearch");
+        ActionListenerOnFailureResponseHandler listenerHandler = new ActionListenerOnFailureResponseHandler();
+        try {
+            transportService.sendRequest(
+                opensearchNode,
+                ExtensionsOrchestrator.REQUEST_ACTION_LISTENER_ON_FAILURE,
```

Comment:
Similar suggestion as this [comment](https://github.com/opensearch-project/OpenSearch/pull/4191#discussion_r943923921), let's change this to `REQUEST_EXTENSION_ACTION_LISTENER_ON_FAILURE`